neo4j/result.go

```diff
@@ -56,11 +55,6 @@ type result struct {
 	delegate ResultWithContext
 }
 
-// deprecated: use newResultWithContext instead
-func newResult(conn db.Connection, str db.StreamHandle, cypher string, params map[string]any) Result {
-	return newResultWithContext(conn, str, cypher, params).legacy()
-}
-
```

Comment:
Is it accessible from the public api?

Reply:
nope (since it starts with a lower case letter, it's not visible outside of the package) and it was actually not used anymore